### ISSUE TITLE
Question about the defintion of Pruning loss

### ISSUE THREAD

**Issue by tanbuzheng** [lychenyoko/content-aware-gan-compression#15]
loss = torch.sum(torch.abs(noisy_img - img_tensor))
I hope to know why define pruning loss like above and the reason for minimizing this loss.

**Comment by lychenyoko** [comment]
First, it's a back-propagate-able loss. Second, we use it to evaluate channels' sensitivity for the content region noises. Please refer to the paper Sec. 3.1.2 for more details.
